src/partials/latestBlock/latestBlock.vue

```diff
@@ -7,24 +7,37 @@
         <ae-button type="exciting" size="small" to="/generations">view all</ae-button>
       </div>
       <div class="generation-data">
-        <template v-if="generation">
+        <template>
```

Comment:
Without `v-if`, `<tamplate>` can be removed.

Reply:
`<template>` it is a wrapping element that does not anything in this case, as far as `v-if` removed. So, I proposing to remove it completely to make markup simpler.